docs/absolute.md

```diff
@@ -308,10 +308,13 @@ A difference of two hours will become 7200 seconds when `largestUnit` is `"secon
 However, a difference of 30 seconds will still be 30 seconds even if `largestUnit` is `"hours"`.
 
 By default, the largest unit in the result is seconds.
-Unlike other Temporal types, months and years are not allowed.
-This is because in the ISO calendar, months and years can be different lengths depending on which month is meant and whether the year is a leap year.
+Weeks, months and years are not allowed, unlike the difference methods of the other Temporal types.
+This is because months and years can be different lengths depending on the calendar in use, which month is meant, and whether the year is a leap year.
 `Temporal.Absolute` is intended to be calendar-independent and therefore free of these ambiguities.
 
+If you do need to calculate the difference between two `Temporal.Absolute`s in years, months, or weeks, then you can eliminate this ambiguity by choosing your starting point by converting to a `Temporal.DateTime`.
+For example, you might decide to base the calculation on your user's current time zone, or on UTC, or on your user's preferred calendar.
```

Comment:
You need both.  You can't set a calendar system without setting a time zone and have this work.

Reply:
Yeah, this was badly worded — I intended (user time zone OR utc) AND (user non-iso calendar OR iso) but condensed it wrong :stuck_out_tongue: